src/js/core/widget/core/Appbar.js

```diff
@@ -584,6 +667,38 @@
 				}
 			};
 
+			/**
+			 * Set expanding option
+			 * @method _setExpandingEnabled
+			 * @param {HTMLElement} element
+			 * @param {boolean} enabled
+			 * @member ns.widget.core.Appbar
+			 * @protected
+			 */
+			prototype._setExpandingEnabled = function (element, enabled) {
```

Comment:
Not used. Please remove it.

Reply:
this is handler for widget.option("expandingEnabled", boolean)